src/common/hooks/useConfig/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState, useEffect } from "react";
+import { AsyncStorage } from "react-native";
+
+export interface Config {
+  network: string;
+}
+
+// eslint-disable-next-line @typescript-eslint/class-name-casing
+export interface useConfig {
+  config: Config;
+  loaded: boolean;
+  setValue: <K extends keyof Config>(key: K, value: Config[K]) => Promise<void>;
+}
```

Comment:
Don't really need to export this. But having it as an interface since it will be used later in the `setValue` function. Will rename it to `ConfigHook` and remove from export.